message-index/messages/GHC-6446/example1/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Do notation in pattern match
+---
+
+When pattern matching, `do` blocks are not allowed as expressions to be pattern matched against.
```

Comment:
Maybe "blocks" isn't the correct term here (See #99)

Reply:
```suggestion
When pattern matching, `do` expressions are not allowed as patterns to be matched against.
```